lib/class/search.class.php

```diff
@@ -1386,6 +1434,157 @@ private function video_to_sql()
     }
 
     /**
+     *  movie_to_sql
+     *
+     * Handles the generation of the SQL for video searches.
+     */
+    private function movie_to_sql()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method movie_to_sql() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.](https://www.codacy.com/app/paul/ampache/file/616134479/issues/source?bid=7631&fileBranchId=2290754#l1441)